ansible/ci-build-collector-full.yml

```diff
@@ -19,7 +19,7 @@
     - name: Build full image
       when:
         - build_full_image
-        - arch != 'arm64'
+        - arch != 'arm64' and arch != 'ppc64le' and arch != 's390x'
```

Comment:
This is the part where we temporarily disable full builds for multi-arch, right?

Reply:
yeah, it is. I'll remove before merging.